### PR TITLE
Add Jay M. proxy voting profile

### PR DESCRIPTION
**Pull request by jaym2518**

### CODE REVIEW

File: proxies.json [hunk 1]
```diff
@@ -14,5 +14,14 @@
...
+  },
+  {
+    "wallet": "84Q9QQ1DeqsbeBMiaKuh2Vk1wrfKirkZXBrTKyLL2apU",
+    "name": "Jay M.",
+    "image": "./proxies/jaym2518/image.png",
+    "description": "Jay M.",
+    "detail": "./proxies/jaym2518/detail.md",
+    "networks": ["HNT", "MOBILE", "IOT"]
   }
+
```
Comment (abhay) [inline proxies.json]:
```suggestion
```

File: proxies.json [hunk 2]
```diff
@@ -14,5 +14,14 @@
...
     "detail": "./proxies/ferebee/detail.md",
     "networks": ["HNT", "MOBILE", "IOT"]
+  },
+  {
+    "wallet": "84Q9QQ1DeqsbeBMiaKuh2Vk1wrfKirkZXBrTKyLL2apU",
+    "name": "Jay M.",
+    "image": "./proxies/jaym2518/image.png",
+    "description": "Jay M.",
+    "detail": "./proxies/jaym2518/detail.md",
+    "networks": ["HNT", "MOBILE", "IOT"]
```
Comment (abhay) [inline proxies.json]: just confirming you want to be assigned in all three networks?

**Review by abhay — COMMENTED** [review]
LGTM. quick question and can merge, @jaym2518.

File: proxies.json (hunk 2 above)
Comment (jaym2518) [reply proxies.json]: Good question. I removed IOT (I think I did, anyway..)

Comment (abhay) [reply proxies.json]: you did it :)

**Pull request merged**